packages/directory/addon/mixins/search-filter.js

```diff
@@ -19,8 +19,6 @@ export default Mixin.create({
    */
   searchResults: computed('directory.q', 'sortedItems', function() {
     let queryString = get(this, 'directory.q');
-    return get(this, 'sortedItems').then(
-      items => (isEmpty(queryString) ? items : searchRecords(items, queryString, 'title'))
-    );
+    return this.sortedItems.then(items => (isEmpty(queryString) ? items : searchRecords(items, queryString, 'title')));
```

Comment:
`(isEmpty(queryString))` => `isEmpty(queryString)`

Reply:
It's a `prettier` thing